### DISCUSSION TITLE
Does it support to get the content by batch api from issues in jira-python?

### DISCUSSION THREAD

**Discussion by Inurogon** [pycontribs/jira#1857] [General]
Hi Teams, Sorry to bother you, we are currently using the enterprise version of the Jira system and would like to develop some tools to search X-ray test cases from the jira system.  At present, We use the interfaces, but due to the need to seach too many issue, the performance cannot be satisfied. It will take more than 15 minutes to call the interface and search for 1000 issues. Do you have any plans to provide bulk interface support to help us solve this problem？The calling interface is as follows:
**jira.get_feilds()**
Besides,I have others issue. Do you have any interface for batch uploading test cases through csv or excel,
Thank a millions.

**Comment by adehad** [comment]
As long as there is an API that Jira provides there will be a way to do it programmatically. You can always use the `JIRA` class provided this repository as a base (and get the community contributed features like the authentication etc) and create your own methods for accessing these other features. It appears at the moment we do not have much support for the X-ray endpoints
